src/components/home/ContactsList.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux';
+import { getAllContacts, setSortStatus } from '../../redux/contacts/actions';
+import { selectContacts, selectIsLoading, selectSortStatus } from '../../redux/contacts/selectors';
+import styles from "./home.module.scss";
+
+const ContactsList = () => {
+  // DISPATCH HOOK
+  const dispatch = useDispatch();
+
+  // SELECTORS
+  const contacts = useSelector(selectContacts);
+  const loading = useSelector(selectIsLoading);
+  const sortStatus = useSelector(selectSortStatus);
+
+  const sortContacts = () => {
+    console.log('hit sortContacts function: ', contacts)
+    console.log('sortStatus at begging of function: ', sortStatus)
+    if (sortStatus === 'first-ascending') {
+      contacts.sort((a: any, b: any) => {
+        console.log('sort firstName a-z: ', contacts)
+        return a.firstName - b.lastName
+      })
+    } else if (sortStatus === 'descending') {
+      contacts.sort((a: any, b: any) => b - a)
+    } else {
+      console.log('sortStatus: ', sortStatus)
+      console.log('else...sortStatus did not match')
+    }
+  }
+
+  const contactsMap = contacts.map((contact, i) => (
+    <div className={styles.contactCard} key={i}>
+      <div>{contact.firstName}</div>
+      <div>{contact.lastName}</div>
+      <div>{contact.phone}</div>
+      <div>{contact.email}</div>
+      <div>{contact.address1}</div>
+      <div>{contact.address2}</div>
+      <div>{contact.city}</div>
+      <div>{contact.state}</div>
+      <div>{contact.zip}</div>
+    </div>
+  ))
+
+
+  return (
+    <>
+      <div className={styles.sortDropdownContainer}>
+        <select value={sortStatus} onChange={(e) => {
```

Comment:
I think it'd be better to make sorting icons that go in the header next to the label. It's common practice to use small triangles. I also think it'd be better to make an enum for the values so you don't have to check against string literals. Something like this:

```
export enum SortValues {
   FIRST_ASC,
   FIRST_DESC,
   LAST_ASC,
   LAST_DESC
}
```